docs/cli-reference/dfx-sns.md

```diff
@@ -0,0 +1,79 @@
+
+Use the `dfx sns` subcommands to simulate decentralizing a dapp.
+
+The basic syntax for running `dfx sns` commands is:
+
+``` bash
+dfx sns [subcommand] [flag]
+```
+
+Depending on the `dfx sns` subcommand you specify, additional arguments, options, and flags might apply. For reference information and examples that illustrate using `dfx sns` commands, select an appropriate command.
+
+| Command                             | Description                                                                   |
+|-------------------------------------|-------------------------------------------------------------------------------|
+| [`create`](#_dfx_sns_create)        | Creates an SNS confoiguration template                                        |
+| [`validate`](#_dfx_sns_validate)    | Checks whether the sns config file is valid.                                  |
+| `help`                              | Displays usage information message for a specified subcommand.                |
+
+To view usage information for a specific subcommand, specify the subcommand and the `--help` flag. For example, to see usage information for `dfx sns validate`, you can run the following command:
+
+``` bash
+dfx sns validate --help
+```
+
+
+## dfx sns create
+
+Use the `dfx sns create` command to create an SNS configuration file.  The configuration file specifies important, legally and financially relevant details about dapp decentralization.  The file leaves blank parameters such as token name; you will need to fill these in.
```

Comment:
```suggestion
Use the `dfx sns create` command to create an SNS configuration file. The configuration file specifies important, legally and financially relevant details about dapp decentralization.  The file leaves blank parameters such as token name; you will need to fill these in.
```

Reply:
haha.  I'm still in the habit of English Spacing (double space after a full stop).  Switzerland uses French Spacing, as do most people now as people have got used to MS Word making the first space after a stop wider.